setup.py

```diff
@@ -17,22 +17,16 @@
       long_description=readme,
       license='MIT License',
       url='http://pythonhosted.org/blinker/',
+      python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*',
```

Comment:
`python_requires` requires `pip>=9.0`.

Reply:
Yes, and if the user has `pip<9.0` then it has no effect.

Reply:
Ah, good, no troubles then. :)